docs/advanced_topics/images/amp.rst

```diff
@@ -219,6 +219,65 @@ For example:
         AMP MODE IS ACTIVE!
     {% endif %}
 
+Using a different page template when AMP mode is active
+-------------------------------------------------------
+
+You're probably not going to want to use the same templates on the AMP site as
+you do on the regular web site. So let's add some logic in so Wagtail will
+automatically use separate whenever a page is served with AMP enabled.
+
+We can implement this logic as a mixin which allows us to implement it on many
```

Comment:
"We can use a mixin, which allows us to re-use the logic on different page types" ?
At the very least drop the "implement .. implement" construct :)

Reply:
Thanks!